packages/loopring_v3/contracts/aux/bridge/Bridge.sol

```diff
@@ -243,6 +255,9 @@ contract Bridge is IBridge, BatchDepositor, Claimable
         // Total amounts transferred to the bridge
         uint[] memory totalAmounts = new uint[](ctx.tokens.length);
 
+        // Verify withdrawals
+        _processWithdrawals(ctx, totalAmounts);
```

Comment:
So yeah these are done at the end because that's how things work on L2:
- First all transfers to the bridge
- Then we are able to withdraw funds from the bridge account

So `_processWithdrawals` after all transfers makes the most sense. Otherwise we wouldn't be able to run over all L2 transactions in order here and we'd have to jump around in `txsData`.

The current implementation also counts on the current order because `totalAmounts` is calculated using the transfers, so it's only correctly to pass this value into `_processWithdrawals` after all `_processConnectorCall` calls.